### PR TITLE
change logger level when connection dropped or timed-out

### PR DESCRIPTION
**Pull request by jaehyoeklee**
just change logger level for sending event to sentry

### CODE REVIEW

**Review by send-leo — APPROVED** [review]
LGTM

File: kazoo/protocol/connection.py [hunk 1]
```diff
@@ -605,9 +605,9 @@ def _connect_attempt(self, host, port, retry):
             return STOP_CONNECTING
         except (ConnectionDropped, KazooTimeoutError) as e:
             if isinstance(e, ConnectionDropped):
-                self.logger.warning('Connection dropped: %s', e)
+                self.logger.exception('Connection dropped: %s', e)
```
Comment (jinwookim0) [inline kazoo/protocol/connection.py]: How about adding the text `Zookeeper` or `Kazoo`? 

Eg. "Zookeeper Connection dropped: {e}"

**Review by jinwookim0 — APPROVED** [review]
👍

File: kazoo/protocol/connection.py (hunk 1 above)
Comment (jaehyoeklee) [reply kazoo/protocol/connection.py]: Sounds good I'll affect

**Pull request merged**